### DISCUSSION TITLE
[Console] Customize configurations of microservices handled by the Console

### DISCUSSION THREAD

**Discussion by frengooo** [mia-platform/community#390] [Q&A] [answered]
### What product are you having troubles with?

Console

### What Console version are you using?

12.1.0

### Description

Hi guys, I need to manually add some environment variables on the `v1-adapter` microservice.
This service is handled by the Console, is there a way to do that?

**Comment by fredmaggiowski** [comment, marked as answer]
Hi,

you should be able to specify them using the `api-console-config/core-services.json` extension; something like this should be enough:

```json
{
    "v1-adapter": {
        "envs": {
            "MY_VAR": "MY_VAL"
        }
    }
}
```

> [!IMPORTANT]
> Please be advised that the v1-adapter, together with the legacy headless CMS, is a deprecated component and should be replaced with the new [Microfrontend composer](https://docs.mia-platform.eu/docs/microfrontend-composer/what-is)-based Backoffice


---


<img width="790" alt="image(3)" src="https://github.com/mia-platform/community/assets/7142570/6c3d256b-9ffe-41a1-b773-868289bc2858">
